--- a/setup.py
+++ b/setup.py
@@ -41,7 +41,7 @@
             ['_festival.cpp'],
             include_dirs=[festival_include, speech_tools_include],
             library_dirs=[festival_lib],
-            libraries=['Festival', 'estools', 'estbase', 'eststring'],
+            libraries=['Festival', 'estools', 'estbase', 'eststring', 'ncurses', 'gomp'],
         ),
     ],
     platforms=["*nix"],
--- a/_festival.cpp
+++ b/_festival.cpp
@@ -1,6 +1,6 @@
 #include <stdio.h>
 #include <festival.h>
-#include "festival_private.h"
+#include <festival_private.h>
 #include "Python.h"
 
 
