--- a/setup.py
+++ b/setup.py
@@ -8,7 +8,7 @@
 here = path.abspath(path.dirname(__file__))

 # Get the long description from the README file
-with open(path.join(here, 'PREADME.txt'), encoding='utf-8') as f:
+with open(path.join(here, 'README.rst'), encoding='utf-8') as f:
     long_description = f.read()

 requirements = [
@@ -16,7 +16,6 @@
 ]
 
 setup_requirements = [
-    'pytest-runner',
 ]
 
 test_requirements = [
